### ISSUE TITLE
1.19.1: Spy not registering until you log off. 

### ISSUE THREAD

**Issue by toxinyx** [kangarko/ChatControl#2050]
### Are you using MySQL?

No

### Are you using BungeeCord?

No

### "/chc debug" output (strongly recommended)

https://ufile.io/yl21zrr8

### Information about the issue/bug

There's not much consistency to it as it's not every staff member, but it both happens for me and one other member of my staff team.

It doesn't register /spy on/off until we relog. E.G I run /spy on, but it doesn't enable spying until I leave: when I rejoin, I'm spying. I run /spy off to turn it off, and the same thing happens. It doesn't turn off, I log off, back on and it's off again.

**Comment by kangarko** [comment]
Thanks, going to test in 24 hours

**Comment by kangarko** [comment]
I was not able to reproduce nor find this problem. Does it always happen to you or is it random? Try removing Essentials for one testing round to see if it can be caused by a plugin conflict. If yes then you need to disable Essentials' spy feature there instead.

**Comment by TheIntolerant** [comment]
To help us confirm if it is chatcontrol or another plugin do you mind sending a screenshot of this if you continue to have this issues?

**Comment by toxinyx** [comment]
Trying now to see what happens :)

**Comment by toxinyx** [comment]
Initially tried disabling Essentials spy right off the bat - nothing changed, it still didn't register until I'd logged off.
To be extra cautious (in case I'd just disabled it wrong), I just removed Essentials from my plugin list and it's still happening. 

So far it has happened to me every time I've tried. As I said in my original post, it doesn't happen to all my staff members and only seems to happen to me & one other.

screenshot requested:
(line where it says spy was successfully disabled, and then what /spy status printed right after)
![image](https://user-images.githubusercontent.com/86858026/184240756-7458a529-d22c-4fb8-ae56-e4ba80e16a24.png)
